src/main/steps/c100-rebuild/child-details/util.ts

```diff
@@ -30,12 +30,43 @@ export const getDataShape = (): ChildrenDetails => ({
   },
 });
 
+export const getOtherChildDataShape = (): OtherChildrenDetails => ({
+  id: uuidv4(),
+  firstName: '',
+  lastName: '',
+  personalDetails: {
+    dateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    isDateOfBirthUnknown: YesNoEmpty.EMPTY,
+    approxDateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    gender: Gender.EMPTY,
+    otherGenderDetails: '',
+  },
+});
+
 export const getChildDetails = (children: ChildrenDetails[] | [], childId: string): ChildrenDetails | undefined =>
   children.find(child => child.id === childId);
 
+export const getOtherChildDetails = (
+  children: OtherChildrenDetails[] | [],
+  childId: string
+): OtherChildrenDetails | undefined => children.find(child => child.id === childId);
+
 export const updateChildDetails = (children: ChildrenDetails[], childDetails: ChildrenDetails): ChildrenDetails[] =>
   children.map(child => (child.id === childDetails.id ? childDetails : child));
 
+export const updateOtherChildDetails = (
```

Comment:
Why we need another method when updateChildDetails can be tweaked to be reused across child-details & other children? Please tweak the updateChildDetails so that it can be reused.